cypress/integration/price/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js

```diff
@@ -56,7 +56,33 @@ let originalPriceList;
 let originalUOM;
 let originalTaxCategory;
 
+it('Read fixture and prepare the names', function() {
+  cy.fixture('price/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.json').then(f => {
+    priceSystemName = appendHumanReadableNow(f['priceSystemName']);
+    priceListName = appendHumanReadableNow(f['priceListName']);
+    priceListVersionName = appendHumanReadableNow(f['priceListVersionName']);
+
+    priceListSchemaVersionName = appendHumanReadableNow(f['priceListSchemaVersionName']);
+    priceListVersion2ValidFrom = f['priceListVersion2ValidFrom'];
+
+    priceListVersionNameSearch1 = new RegExp(f['priceListVersionNameSearch1']);
+    priceListVersionNameSearch2 = new RegExp(f['priceListVersionNameSearch2']);
+
+    categoryName = appendHumanReadableNow(f['categoryName']);
+    productName1 = appendHumanReadableNow(f['productName1']);
+    productName2 = appendHumanReadableNow(f['productName2']);
+    productType = f['productType'];
+
+    priceListSchemaName = appendHumanReadableNow(f['priceListSchemaName']);
+    surchargeAmount = f['surchargeAmount'];
+  });
+});
+
 describe('Create Price and Products', function() {
+  it('reaaaaaalyyyyyyyyy', function() {
+    cy.wait(5000);
+  });
+
```

Comment:
This looks a bit ugly:( do we reaaaalyyy need the wait in here?

Reply:
Unfortunately, sometimes (eg. 2/5 maybe) we get that problem with `getLanguageSpecific`, so this wait is needed right now -- at least on my machine :(

Reply:
ok